src/components/select/multi-select/multi-select.stories.mdx

```diff
@@ -132,6 +132,49 @@ You can use `listPlacement` prop to set the position of the select list relative
   </Story>
 </Canvas>
 
+#### Controlled with custom onChange
```

Comment:
I think it would be better if we just add unit and automation tests (I'm not sure is Simple and Filterable have them already) for each select component than documenting in storybook it as it's a pattern we advise against using. The linked issue has a codesandbox so QA will be able to use that to test.